test/testutil.lua

```diff
@@ -22,13 +22,6 @@ local M = {
   paths = Paths,
 }
 
---- @param p string
---- @return string
-local function relpath(p)
-  p = vim.fs.normalize(p)
-  return (p:gsub('^' .. uv.cwd, ''))
```

Comment:
This is an interesting impl, maybe we should try it? But make sure that `cmd` ends with a slash :P

Reply:
I'm pretty sure I wrote and that it doesn't work. I half-assed it when I wrote it because we weren't using it anyway (it's currently gated behind a check that is set to false).

Reply:
The tests will tell us :)

May also need `normalize(uv.cwd)`. And technically un-normalize the result, but that's a simple matter of counting slashes.

Reply:
> Would be very nice if we can delegate all the heavy lifting to normalize, for as many purposes as possible.

Sure, if possible we should do that. Need to figure out what the "correct"/our intended behavior we wanna be, let me know if you have any ideas.

Reply:
That's basically [this comment](https://github.com/neovim/neovim/pull/31790#issuecomment-2565799250) and has the drawbacks of not working for path outside of target directory.

Reply:
Maybe needs wrapping with `vim.pesc()` for good measure since cwd may contain `.`.